src/stateless/ChipWithMultiselect.vue

```diff
@@ -71,6 +75,9 @@ export default {
         selectionChange (selected) {
             this.$emit('input', selected)
         },
+        searchChange: debounce(function (searchQuery) {
```

Comment:
Why do we need debounce here?

Reply:
Silly me, copied from `mab` where I assumed I'll even sync search queries to URL query params. Will remove, tnx!